### PR TITLE
{vis}[foss/2016b] OpenCV 3.1.0 (REVIEW)

### PR DESCRIPTION
**Pull request by hajgato**
created from intel/2016b and older foss configs.
needs PR ~~#4119~~

### CODE REVIEW

File: easybuild/easyconfigs/o/OpenCV/OpenCV-3.1.0-foss-2016b.eb [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    ('LibTIFF', '4.0.6'),
+    ('JasPer', '1.900.1'),
+    ('Java', '1.8.0_92', '', True),
+    ('ant', '1.9.7', '-Java-%(javaver)s', True),
+    ('GLib', '2.49.5'),
+    ('GTK+', '2.24.31'),
+]
+
+#The destination directory is "linux-<MD5>" see 3rdparty/ippicv/downloader.cmake
+ppicv_dir = '3rdparty/ippicv/downloads/linux-808b791a6eac9ed78d32a7666804320e/'
```
Comment (boegel) [inline easybuild/easyconfigs/o/OpenCV/OpenCV-3.1.0-foss-2016b.eb]: @hajgato typo, missing leading `i`, which leads to `NameError: name 'ippicv_dir' is not defined`

Comment (hajgato) [inline easybuild/easyconfigs/o/OpenCV/OpenCV-3.1.0-foss-2016b.eb]: @boegel: Oops, corrected

**Comment by boegel** [comment]
lgtm

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 8 out of 8 (8 easyconfigs in this PR)
node2116.delcatty.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/49ac0b9c24937dba2530b06a23325709 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 9 out of 9 (8 easyconfigs in this PR)
node2494.golett.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/a5fa8b0ec757ba82fe2fa0668248262e for a full test report.

**Comment by boegel** [comment]
Going in, thanks @hajgato!

**Pull request merged**